ros/api/v0/recommendations.py

```diff
@@ -53,7 +53,34 @@ def get(self, host_id):
                 if rule_data:
                     rule_dict = rule_data.__dict__
                     recommendation = {}
+                    instance_price = 0
+                    candidates = rule_hit.get('details').get('candidates')
+                    summaries = rule_hit.get('details').get('summary')
+                    instance_price += rule_hit.get('details').get('price')
```

Comment:
What if `price` key not present? Also could you share why you are adding it with `0`? Can't we directly use it?

Reply:
Price will always be there I guess. Otherwise `get` method will throw an error. But if we use brackets here (`rule_hit['details']`) then it gives an error-> int needed str found. That's why used `get` here

Reply:
`get` method won't throw an error but adding that value to instance price failed with an exception.